models/cam-fv/readme.rst

```diff
@@ -1,208 +1,314 @@
-
-The 19 public interfaces in ``model_mod`` are standardized for all DART
-compliant models. These interfaces allow DART to get the model state and
-metadata describing this state, find state variables that are close to a given
-location, and do spatial interpolation for a variety of variables required by
-observational operators.
+.. code-block::
 
-Namelist
---------
+   cesm2_1/                             Directory containing scripts developed for CESM2_1
+       spinup_single                     Setup a single instance (member) CAM-FV case to advance a model state 
+                                         some months to a desired date.
+       setup_hybrid                      Basic script to set up an assimilation case.
+          DART_config.template           Modified to create the script which modifies a CESM CASE to do assimilation.
+             no_assimilate.csh.template  Modified to create a script which does no assimilation, 
+                                         but prepares files for the next model advance.
+             assimilate.csh.template     Modified to create the assimilate.csh script
+       compress.csh                      Example of compressing assimilation output for efficient archiving.
+                                         Can be called by assimilate.csh
+       mv_to_campaign.csh                Example of how to use globus to move files to a remote archive.
+       setup_advanced                    Like setup_hybrid, but more model and assimilation features can be modified.
+                                         It modifies DART_config.template like setup_hybrid does.
+       setup_single_from_ens             Set up a single-instance run using initial conditions taken from 
+                                         a single instance of a multi-instance CAM hindcast.  Useful for debugging.
+       standalone.pbs                    Batch job tests of assimilation with no model advances.
+          test_assimilate.csh            A simpler (earlier) form of assimilate.csh.  
+    cesm2_0/                             Similar contents to cesm2_1, plus the following.
+       obs_seq_tool_series.csh           Script to process a series of obs_seq.final files,
+                                         to change any of the properties available to obs_sequence_tool.
+       spinup_single_sst.25              Same as cesm2_1/spinup_single, but uses a high resolution SST dataset.
+ 
+The scripts  in cesm#_# will handle, for that CESM version;
+
+   * all CAM-FV "physics" variants and vertical resolutions.
+     For example, CAM5.5, CAM6, ..., WACCM4, WACCM6, WACCM-X, ..., CAM-Chem.
+   * all horizontal resolutions of CAM-FV; 1.9x2.5 (f19xf19), 0.9x1.25 (f09xf09), ....
+
+Physics variants of other dycores are handled in other "model" interfaces,
+such as models/cam-se.
+
+.. _SourceMods: 
+.. _reanalysis: https://github.com/kdraeder/cesm}{github.com/kdraeder/cesm
```

Comment:
Comments or documentation?

Reply:
These are supposed to be targets for links.  
I haven't been able to test them yet, but they seem to show up as noncomments in atom and vim.